.github/workflows/matchers/actionlint.json

```diff
@@ -0,0 +1,17 @@
+{
+  "problemMatcher": [
+    {
+      "owner": "actionlint",
+      "pattern": [
+        {
+          "regexp": "^(?:\\x1b\\[\\d+m)?(.+?)(?:\\x1b\\[\\d+m)*:(?:\\x1b\\[\\d+m)*(\\d+)(?:\\x1b\\[\\d+m)*:(?:\\x1b\\[\\d+m)*(\\d+)(?:\\x1b\\[\\d+m)*: (?:\\x1b\\[\\d+m)*(.+?)(?:\\x1b\\[\\d+m)* \\[(.+?)\\]$",
```

Comment:
I hope you didn't summon a golem or something with this incantation.

Reply:
I just used the json provided by the actionlint action.

https://github.com/rhysd/actionlint/blob/main/docs/usage.md#problem-matchers

Reply:
👍 and completely missed my joke..._fine_.